package.json

```diff
@@ -25,28 +49,6 @@
   "author": "",
   "license": "Apache V2",
   "devDependencies": {
-    "async": "~0.2.6",
-    "grunt": "~0.4.1",
```

Comment:
these are all `devDependencies`, we don't need any of these to run Fauxton in production, just to prepare the build.